dts/bindings/clock-management/clock-source.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Generic clock source that may be gated. This node accepts one specifier,
+  which either gates the clock (when set to 0), or ungates the clock
+  (when nonzero)
+  Other specifier values may cause undefined behavior.
+
+compatible: "clock-source"
+
+include: [clock-node.yaml]
+
+properties:
+  reg:
+    required: true
+    description: |
+      Tuple containing address of register and width of the bitfield
+      to set to enable the clock source
+
+  frequency:
```

Comment:
this is called `frequency` which is rather inconsistent with the other similar props which are named `clock-frequency`

Reply:
Fixed